forge-gui/res/cardsfolder/upcoming/siege_smash.txt

```diff
@@ -0,0 +1,8 @@
+Name:Siege Smash
+ManaCost:1 R
+Types:Instant
+K:Split second
+A:SP$ Charm | Choices$ DBDestroy,DBPump
+SVar:DBDestroy:DB$ Destroy | ValidTgts$ Artifact | TgtPrompt$ Select target artifact. | SpellDescription$ Destroy target artifact.
+SVar:DBPump:DB$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature to get +3/+2 and trample | NumAtt$ +3 | NumDef$ +2 | KW$ Trample | SpellDescription$ Target creature gets +3/+2 and gains trample until end of turn.

```

Comment:
Probably just my OCD but I say get rid of `TgtPrompt` from these lines... the one having a `.` and the other not...